packages/sdk/src/swap/sdk.ts

```diff
@@ -494,7 +494,7 @@ export class SwapSDK {
         channelOpeningFee: result.channelOpeningFee,
       };
     } else {
-      response = await this.trpc.openSwapDepositChannel.mutate(depositAddressRequest);
+      response = await this.trpc.openSwapDepositChannel.mutate({ ...depositAddressRequest, quote });
```

Comment:
decided to pass the full quote bc we wont need an sdk update if we want to store additional data like quoted fees in the future